data/human/human missions.txt

```diff
@@ -6890,3 +6890,109 @@ mission "Hogshead's Dilemma II"
+			``
+			`	"That's odd," you say to Caesar. "How did you get lost on the wrong planet, huh boy?"`
+			`	You pet Caesar, who seems glad of the attention, and decide what to do.`
+			choice
+				`	(Take Caesar to his owners on Hestia.)`
+					goto takehim
+				`	(Leave Caesar here.)`
+
+			`	Perhaps his owners are on vacation and will be back looking for him any minute. Although it's getting dark and no one seems to be around, you decide the best course of action to leave Caesar at the park.`
+			`	As you walk off you hear Caesar whining, and as you look back you notice him slink off into the park. But you've made your decision; hopefully everything will work out for the best.`
+				decline
+
+			label takehim
+			`	"Well, come on boy. Let's go." Caesar eagerly follows you back to your ship. He seems greatly relieved to be with you, and not left all alone at the park.`
+			`	Once back on your ship, you make sure he's fed, watered, and settled in for his interstellar journey.`
+				accept
+
+	on complete
+		payment 1000
+		conversation
+			branch "transported pookie"
+				has "Pookie, Part 2: done"
+				not `outfit: "Puny"`
+			`Caesar has been very obedient the whole trip, much to your convenience. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads "Mitchell Ranch."`
+				goto ranch
+
+			label "transported pookie"
+			`Caesar has been very obedient the whole trip, unlike some other certain dogs you've transported in the past. He lays down by the airlock while your ship sets down in the local Avalonia spaceport, which is enclosed on all sides by a squalid urban landscape. The honeycombed homes of humble help. You rent a vehicle and take yourself and Caesar beyond the blighted city and out into the pleasant picture-perfect paradise of the wealthy. One princely estate after another rolls past you as you drive deeper into the rural domains of the Republic's super-wealthy. Eventually, you come across a large metal gate that reads, "Mitchell Ranch."`
+
+			label ranch
+			`	It's another forty-five minutes of driving down long winding roads, past various outbuildings, lodges, retreats, guest houses, and hobby barns, before at long last you arrive at the massive central mansion. Waiting at the bottom of stone stairs is a footman who opens the door for you and summons the senior butler.`
+			`	The butler arrives and looks you over unhappily. "The lady of the manor is not expecting... guests. Perhaps, you've misread your appointment, or more likely, arrived at the wrong estate."`
```

Comment:
This reads a little awkwardly, with the "or more likely" not being separated by a comma. I'm not sure how you'd fix it, though, since adding a comma there would make the sentence very choppy.